requirements.txt

```diff
@@ -18,20 +23,44 @@ flake8-print==3.1.0
 flake8-quotes==2.0.1
 flake8-string-format==0.2.3
 flake8-todo==0.7
-flake8-annotations-complexity==0.0.1
 flake8-variables-names==0.0.1
-flake8-class-attributes-order==0.0.2
-flake8-broken-line==0.1.0
+google-api-python-client==1.7.11
+google-auth==1.6.3
+google-auth-httplib2==0.0.3
+google-auth-oauthlib==0.4.0
+httplib2==0.14.0
+idna==2.8
+importlib-metadata==0.23
+Markdown==3.1.1
+mccabe==0.6.1
+more-itertools==7.2.0
 mypy==0.701
 mypy-extensions==0.4.1
-typing==3.6.4
-typing-extensions==3.7.2
+oauthlib==3.1.0
+packaging==19.2
 pep8-naming==0.7.0
+pluggy==0.13.0
+py==1.8.0
+pyasn1==0.4.7
+pyasn1-modules==0.2.7
+pycodestyle==2.5.0
+pydocstyle==3.0.0
+pyflakes==2.1.1
+pyparsing==2.4.2
 pytest==4.2.1
 pytest-cov==2.5.1
-attrs==19.1.0
+requests==2.22.0
+requests-oauthlib==1.2.0
 rozental-as-a-service==0.1.0
-google-api-python-client==1.7.11
-google-auth==1.6.3
-google-auth-httplib2==0.0.3
-google-auth-oauthlib==0.4.0
+rsa==4.0
+six==1.12.0
+snowballstemmer==2.0.0
+soupsieve==1.9.4
+tabulate==0.8.3
+typed-ast==1.3.5
+typing==3.6.4
+typing-extensions==3.7.2
+uritemplate==3.0.0
+urllib3==1.25.6
+wcwidth==0.1.7
+zipp==0.6.0
```

Comment:
This changes are not required to write tests. Move them to separate pr from here, so we could merge this.

Reply:
Commit reverted